simpleflow/swf/process/decider/base.py

```diff
@@ -233,3 +229,23 @@ def decide(self, decision_response, task_list):
             decisions = [decision]
 
         return decisions
+
+
+def process_decision(poller, decision_response):
+    logger.info("taking decision for workflow {}".format(poller._workflow_name))
```

Comment:
minor: maybe log the pid? (DEBUG level?)

Reply:
We log in the "spawn()" method ; I tried to keep something as similar as the "worker/base.py" code so we can factor this at some point.

Reply:
But we log the parent's, not the child's pid

Reply:
Ah, good point, ok. I'll add this in both places in debug level then.